bindings/py/cpp_src/plugin/PyBindRegion.hpp

```diff
@@ -74,6 +74,12 @@ namespace nupic
         void deserialize(BundleIO& bundle) override;
 
 
+		    bool operator==(const RegionImpl &other) const override {return true;}
```

Comment:
I'm worried this will once give somebody headache. Can you NTA_THROW there?

Reply:
>I'm worried this (compare on PyBindRegion) will once give somebody headache. Can you NTA_THROW there?

Good idea.  I will do that.